var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -80,6 +81,7 @@ class Rocblas(CMakePackage):
         "6.0.2",
         "6.1.0",
         "6.1.1",
+        "6.1.2",
```

Comment:
in the rocblas package, can you move  depends_on("rocm-cmake", type="build")
 inside the for loop.

Reply:
brought rocm-cmake into loop